python/ray/ml/train/integrations/rl/rl_trainer.py

```diff
@@ -0,0 +1,201 @@
+from typing import Optional, Dict, Type, Union, Callable, Any
+
+from ray.ml.checkpoint import Checkpoint
+from ray.ml.config import ScalingConfig, RunConfig
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.trainer import Trainer, GenDataset
+from ray.rllib.agents.trainer import Trainer as RLLibTrainer
+from ray.rllib.utils.typing import PartialTrainerConfigDict, EnvType
+from ray.tune import Trainable, PlacementGroupFactory
+from ray.tune.logger import Logger
+from ray.tune.registry import get_trainable_cls
+from ray.tune.resources import Resources
+from ray.util.annotations import PublicAPI
+from ray.util.ml_utils.dict import merge_dicts
+
+
+@PublicAPI(stability="alpha")
+class RLTrainer(Trainer):
+    """Reinforcement learning trainer.
+
+    This trainer provides an interface to RLLib trainables.
+
+    If datasets and preprocessors are used, they can be utilized for
+    offline training, e.g. using behavior cloning. Otherwise, this trainer
+    will use online training.
+
+    Args:
+        algorithm: Algorithm to train on. Can be a string reference,
+            (e.g. ``"PPO"``) or a RLLib trainer class.
+        scaling_config: Configuration for distributed training, e.g. number
+            of workers or resources per worker.
+        run_config: Run config passed to ``Tuner()``
+        datasets: If specified, datasets used for offline training. Will be
+            configured as an RLLib ``input`` config item.
+        preprocessor: If specified, preprocessors to be applied to the
+            datasets before loading for input training.
+        resume_from_checkpoint: Optional checkpoint to resume training from.
+        **train_kwargs: Additional kwargs.
```

Comment:
What are these kwargs used for? Can we add that in the docstring?

Reply:
Removed for now as currently unused